doc/user/flycheck-versus-flymake.rst

```diff
@@ -315,33 +230,32 @@ to set this variable interactively.
 Multiple syntax checkers per buffer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Flymake** can only use a single syntax checker per buffer.  Effectively, the
-user can only use a single tool to check a buffer, for instance either PHP Mess
-Detector or PHP CheckStyle.  Third party extensions to Flycheck work around this
-limitation by supplying custom shell scripts to call multiple syntax checking
-tools at once.
+**Flymake** will use all the backends added to the
+`flymake-diagnostic-functions` hook to check a buffer; all backends are started
```

Comment:
"Also, backends can report multiple times per call, which is nice for performance and eventually writing a backend that replaces flyspell with flymake. And also it's also possible to first focus on the visible regions of the buffer and then the invisible regions."

Reply:
> backends can report multiple times per call,

That's very nice indeed. 

> t's also possible to first focus on the visible regions of the buffer 

Are there any backends that currently do that?

Reply:
> Are there any backends that currently do that?

No.